src/main/java/com/solvd/micro9/vkontaktah/persistence/UserRepository.java

```diff
@@ -0,0 +1,17 @@
+package com.solvd.micro9.vkontaktah.persistence;
+
+import com.solvd.micro9.vkontaktah.domain.User;
+import org.springframework.data.neo4j.repository.Neo4jRepository;
+import org.springframework.data.neo4j.repository.query.Query;
+
+public interface UserRepository extends Neo4jRepository<User, String> {
+
+    @Query("CREATE CONSTRAINT user_id_unique IF NOT EXISTS "
```

Comment:
@eyakauleva I beleive that it is part of `DDL` (data definition language) like create a table, create an index, and so on, in code we should only do `DML` (data manipulation language) like insert, match, delete, etc.

Reply:
@h1alexbel so to create index and etc. we should do that manually in database?

Reply:
@eyakauleva yes, it's like schema creation in relational databases

you can try to find some automated tools for that, like liquibase for postgres

Reply:
done 58f975b